application/controllers/Sstv.php

```diff
@@ -0,0 +1,95 @@
+<?php if ( ! defined('BASEPATH')) exit('No direct script access allowed');
+
+/*
+	Controller for SSTV Images
+*/
+
+class Sstv extends CI_Controller {
+
+    function __construct() {
+        parent::__construct();
+        $this->lang->load('qslcard');
+        $this->load->model('user_model');
+        if(!$this->user_model->authorize(2)) { $this->session->set_flashdata('notice', 'You\'re not allowed to do that!'); redirect('dashboard'); }
+    }
+
+    public function uploadSSTV() {
+        $this->load->model('user_model');
+        if(!$this->user_model->authorize(2)) { $this->session->set_flashdata('notice', 'You\'re not allowed to do that!'); redirect('dashboard'); }
+
+        if (!file_exists('./assets/sstvimages')) {
+            mkdir('./assets/sstvimages', 0755, true);
+        }
+        $qsoid = $this->input->post('qsoid');
+        
+        $results = array();
+        if (isset($_FILES['sstvimages']) && $_FILES['sstvimages']['error'][0] == 0)
+        {
+            for($i=0; $i<count($_FILES['sstvimages']['name']); $i++) {
+                $file = array(
+                    'name' => $_FILES['sstvimages']['name'][$i],
+                    'type' => $_FILES['sstvimages']['type'][$i],
+                    'tmp_name' => $_FILES['sstvimages']['tmp_name'][$i],
+                    'error' => $_FILES['sstvimages']['error'][$i],
+                    'size' => $_FILES['sstvimages']['size'][$i]
+                );
+                $result = $this->uploadSSTVImage($qsoid, $file);
+                array_push($results, $result);
+            }
+        }
+
+        header("Content-type: application/json");
+        echo json_encode($results);
+    }
+
+    function uploadSSTVImage($qsoid, $file) {
+        $config['upload_path']          = './assets/sstvimages';
+        $config['allowed_types']        = 'jpg|gif|png|jpeg|JPG|PNG|bmp';
```

Comment:
added `bmp` as an option here since thats the default that MMSSTV saves as.   Happy to remove it if there was a specific reason it wasn't added for QSL cards